templates/application-descheduler.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: descheduler
```

Comment:
can we call it glueops-descheduler? As this name lives in the same namespace as the customer facing apps. I think there could be a conflict later. `glueops-` is pretty generic as well but we have already used it for some of the other argocd applications.